python/tvm/_ffi/node_generic.py

```diff
@@ -86,7 +103,7 @@ def const(value, dtype=None):
     value : int or float
         The input value
 
-    dtype : str
+    dtype : str or None, optional
```

Comment:
Not sure which one would be better
```suggestion
    dtype : Optional[str]
```